include/mega/logging.h

```diff
@@ -156,16 +161,19 @@ class SimpleLogger
     using NumBuf = std::array<char, 24>;
     const char* filenameStr;
     int lineNum;
+    int outputted = 0;
+    static const int immediate_threshold = 1024;
+    bool immediate = false;
```

Comment:
Can we name our variables according to our style as `mMember`?

Reply:
sure!